frontend/src/scenes/setup/PreflightCheck.js

```diff
@@ -0,0 +1,236 @@
+
+    const checks = [
+        {
+            id: 'database',
+            name: 'Database (Postgres)',
+            status: preflight.db,
+        },
+        {
+            id: 'backend',
+            name: 'Backend server (Django)',
+            status: preflight.django,
+        },
+        {
+            id: 'redis',
+            name: 'Queue processing (Redis)',
+            status: preflight.redis,
+        },
+        {
+            id: 'frontend',
+            name: 'Frontend built (Node)',
+            status: true, // If this code is run, the front-end is already built
+        },
+        {
+            id: 'tls',
+            name: 'SSL/TLS certificate',
+            status: window.location.protocol === 'https:',
+            caption:
+                state.mode === 'Experimentation'
+                    ? 'Not required for testing / experimenting'
+                    : 'Required, but may be installed later',
+            failedState: state.mode === 'Experimentation' ? 'not-required' : 'warning',
+        },
+    ]
+
+    const runChecks = () => {
+        // Clear the previous result first and add the timeout to show the loading animation
+        loadPreflightSuccess({})
+        setTimeout(() => loadPreflight(), 1000)
+    }
+
+    const handleModeChange = (mode) => {
+        setState({ ...state, mode })
+        if (mode) {
+            runChecks()
+            localStorage.setItem('preflightMode', mode)
+        } else {
+            localStorage.removeItem('preflightMode')
+        }
+    }
+
+    const handlePreflightFinished = () => {
+        window.location.href = '/setup_admin'
+    }
+
+    useEffect(() => {
+        const mode = localStorage.getItem('preflightMode')
+        if (mode) handleModeChange(mode)
+    }, [])
+
+    return (
+        <>
+            <Space direction="vertical" className="space-top" style={{ width: '100%' }}>
+                <h1 className="title text-center" style={{ marginBottom: 0 }}>
+                    Welcome to PostHog!
+                </h1>
+                <div className="page-caption text-center">Understand your users. Build a better product.</div>
+            </Space>
+            <Col xs={24} style={{ marginTop: 60 }}>
+                <h2 className="subtitle text-center space-top">
+                    We're glad to have you here! We'll now guide you to setting up PostHog.
+                </h2>
+            </Col>
+            <Row style={{ display: 'flex', justifyContent: 'center' }}>
+                <div style={{ display: 'flex', alignItems: 'center', flexDirection: 'column' }}>
+                    <img src={hedgehogBlue} style={{ maxHeight: '100%' }} />
+                    <Button type="default" data-attr="support" data-source="preflight">
+                        <a href="https://posthog.com/support" target="_blank" rel="noreferrer">
+                            Support
+                        </a>
+                    </Button>
```

Comment:
![Support](https://user-images.githubusercontent.com/4550621/90756861-2420ca80-e2dd-11ea-84cf-3ef88ae166d0.png)
Just "Support" is kind of ambiguous and not very human. Maybe
```HTML
Got any questions?

<Button>Find support</Button>
```